### PR TITLE
Reduce the image size of Student Ambassador page

### PR DESCRIPTION
**Pull request by nimeshkavindu**
## Purpose
Reduce the image size of Student Ambassador page #1479

## Goals
Reducing the image size of Student Ambassador page

## Approach
Reduced the main image size of Student Ambassador page.

### Screenshots
OnPaste 20240114-191307

  
### Preview Link
https://pr-1610-sef-site.surge.sh/student-ambassador.html
##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Hi, @nimeshkavindu so some changes to be changed.

1. In the Pull Request template you have to put the issue number at `The purpose of this PR is to fix #IssueNumber`
2. At the goals section you have to put the goal of the PR. (Eg: Reducing the image size of Student Ambassador page)
3. In the Approach section you have to put how you did this like the steps you have done.
4. At the preview link section https://pr-{pr_number}-sef-site.surge.sh/ it is like this no so you have to remove this thing {pr_number} and replace the pull request number there and at the last add the folder/file name like https://pr-XXXX-sef-site.surge.sh/student-ambassador.html..
5. At the screenshot section you have to put a screenshot taken in the preview of the change you have done (Eg: The screenshot of the reduced image with whole page)

File: index.html [hunk 1]
```diff
@@ -161,6 +161,7 @@ <h5 class="display-3 text-center project-stat">100+</h5>
                             </p>
                             <h5 class="display-3 text-center project-stat">275 000+</h5>
                             <p class="text-dark text-center stats">
+
```
Comment (YoshithaRathnayake) [inline index.html]: Remove this extra line @nimeshkavindu 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by nimeshkavindu** [comment]
> Hi, @nimeshkavindu so some changes to be changed.
> 
> 1. In the Pull Request template you have to put the issue number at `The purpose of this PR is to fix #IssueNumber`
> 2. At the goals section you have to put the goal of the PR. (Eg: Reducing the image size of Student Ambassador page)
> 3. In the Approach section you have to put how you did this like the steps you have done.
> 4. At the preview link section [https://pr-{pr_number}-sef-site.surge.sh/](https://pr-%7Bpr_number%7D-sef-site.surge.sh/) it is like this no so you have to remove this thing {pr_number} and replace the pull request number there and at the last add the folder/file name like https://pr-XXXX-sef-site.surge.sh/student-ambassador.html..
> 5. At the screenshot section you have to put a screenshot taken in the preview of the change you have done (Eg: The screenshot of the reduced image with whole page)

Done.

File: index.html (hunk 1 above)
Comment (nimeshkavindu) [reply index.html]: > Remove this extra line @nimeshkavindu

Should I create a new commit for this or can I amend the commit?

Comment (YoshithaRathnayake) [reply index.html]: Just click commit suggestion button its enough for now @nimeshkavindu

**Comment by YoshithaRathnayake** [comment]
> > Hi, @nimeshkavindu so some changes to be changed.
> > 
> > 1. In the Pull Request template you have to put the issue number at `The purpose of this PR is to fix #IssueNumber`
> > 2. At the goals section you have to put the goal of the PR. (Eg: Reducing the image size of Student Ambassador page)
> > 3. In the Approach section you have to put how you did this like the steps you have done.
> > 4. At the preview link section [https://pr-{pr_number}-sef-site.surge.sh/](https://pr-%7Bpr_number%7D-sef-site.surge.sh/) it is like this no so you have to remove this thing {pr_number} and replace the pull request number there and at the last add the folder/file name like https://pr-XXXX-sef-site.surge.sh/student-ambassador.html..
> > 5. At the screenshot section you have to put a screenshot taken in the preview of the change you have done (Eg: The screenshot of the reduced image with whole page)
> 
> Done.

Great! @nimeshkavindu 
Do this https://github.com/sef-global/sef-site/pull/1610#discussion_r1451749540

File: index.html (hunk 1 above)
Comment (nimeshkavindu) [reply index.html]: Ok, Thanks @YoshithaRathnayake

**Comment by YoshithaRathnayake** [comment]
Can you create a new branch and to the requested changes and submit the PR again. Because its not good to do it in the master branch. You have done it in the master branch? @nimeshkavindu

**Review by YoshithaRathnayake — APPROVED** [review]
Good job!
Thank you @nimeshkavindu

**Pull request merged**